python/lsst/dm/csc/base/archive_controller.py

```diff
@@ -169,10 +169,10 @@ def build_health_ack_message(self, incoming_msg):
         return d
 
     def construct_send_target_dir(self, target_dir):
-        today = datetime.datetime.now()
-        day_string = today.date()
+        observing_time = datetime.datetime.now(datetime.timezone.utc)  - datetime.timedelta(hours=12)
```

Comment:
There's an extra space here before the `-`

Reply:
Fixed